tunneling/python/app.py

```diff
@@ -0,0 +1,35 @@
+import json
+import urllib
+
+import flask
+import requests
+from flask_cors import CORS
+
+app = flask.Flask(__name__)
+CORS(app)
+
+
+@app.route("/bugs", methods=["POST"])
+def bugs():
+    sentry_host = "sentry.io"
+    known_project_ids = ["123456"]
+
+    envelope = flask.request.data.decode("utf-8")
+    piece = envelope.split("\n")[0]
+    header = json.loads(piece)
+    dns = urllib.parse.urlparse(header.get("dsn"))
```

Comment:
```suggestion
    dsn = urllib.parse.urlparse(header.get("dsn"))
```

Typo `dns` -> `dsn` (here and usages below)